scss/tech.scss

```diff
@@ -1,44 +1,64 @@
+.tech-container {
+  display: grid;
+  grid-template-columns: 200px 200px 200px;
+  grid-template-rows: 300px 300px;
+  padding-left: 20px;
+  font-family: 'Roboto', sans-serif;
+  justify-content: center;
+  grid-gap: 30px;
+}
+
+.primary-item {
+  grid-row: 1/3;
+  grid-column: 1/2;
+  border-right: 2px solid $green;
+  justify-self: center;
+  padding: 0 10px;
+  height: 100%;
+}
+
+.item-1 {
```

Comment:
Remember that the only tech card that is different is the first one, the other ones use the same 'style'.